projects/distribution/distribution/Makefile

```diff
@@ -0,0 +1,45 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=distribution
+REPO_OWNER=distribution
+
+BINARY_TARGET_FILES=registry
+SOURCE_PATTERNS=./cmd/registry
+
+VERSION=$(shell git -C $(REPO) describe --match 'v[0-9]*' --dirty='.m' --always)
+REVISION=$(shell git -C $(REPO) rev-parse HEAD)$(shell if ! git -C $(REPO) diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
```

Comment:
Do these commands return the same output for a given tag, i.e., do they affect build reproducibility?

Reply:
no, they won't